tests/phpunit/S3WriterExceptionTest.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\S3Writer\Tests;
+
+use Aws\Command;
+use Aws\S3\Exception\S3Exception;
+use GuzzleHttp\Exception\ClientException;
+use GuzzleHttp\Psr7\Response;
+use GuzzleHttp\Psr7\Stream;
+use Keboola\Component\UserException;
+use Keboola\S3Writer\S3WriterException;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+
+class S3WriterExceptionTest extends TestCase
+{
+    public function testException(): void
```

Comment:
Tohle bych klidně otestoval bez mocků. Pokud to jde...

Reply:
Asi to k tomu přidám, ale mocky bych tam nechal